scripts/create_sql_lite_dump.py

```diff
@@ -0,0 +1,125 @@
+import click
+import click_config_file
+import isb_lib.core
+from isamples_metadata import Transformer
+from isb_lib.models.isb_core_record import ISBCoreRecord
+from isb_web.isb_solr_query import ISBCoreSolrRecordIterator
+import requests
+from sqlmodel import SQLModel, create_engine, Session
+import logging
+
+# Use this to map between the solr column names and the sqlite column names and control what is
+# written to the dump.
+SOLR_TO_SQLITE_FIELD_MAPPINGS = {
+    "id": "id",
+    "isb_core_id": "isb_core_id",
+    "indexUpdatedTime": "isb_updated_time",
+    "sourceUpdatedTime": "source_updated_time",
+    "label": "label",
+    "searchText": "search_text",
+    "description": "sample_description",
+    "hasContextCategory": "context_categories",
+    "hasMaterialCategory": "material_categories",
+    "hasSpecimenCategory": "specimen_categories",
+    "keywords": "keywords",
+    "informalClassification": "informal_classification",
+    "producedBy_isb_core_id": "produced_by_isb_core_id",
+    "producedBy_label": "produced_by_label",
+    "producedBy_description": "produced_by_description",
+    "producedBy_hasFeatureOfInterest": "produced_by_feature_of_interest",
+    "producedBy_responsibility": "produced_by_responsibility",
+    "producedBy_resultTime": "produced_by_result_time",
+    "producedBy_samplingSite_description": "produced_by_sampling_site_description",
+    "producedBy_samplingSite_label": "produced_by_sampling_site_label",
+    "producedBy_samplingSite_location_elevationInMeters": "produced_by_sampling_site_elevation_in_meters",
+    "producedBy_samplingSite_location_ll": "produced_by_sampling_site_location_ll",
+    "producedBy_samplingSite_placeName": "produced_by_sampling_site_place_name",
+    "registrant": "registrant",
+    "samplingPurpose": "sampling_purpose",
+    "curation_label": "curation_label",
+    "curation_description": "curation_description",
+    "curation_accessConstraints": "curation_access_constraints",
+    "curation_location": "curation_location",
+    "curation_responsibility": "curation_responsibility",
+    "relatedResource_isb_core_id": "related_resources_isb_core_id",
+    "source": "source"
+}
+
+NUMERIC_COLUMNS = ["produced_by_sampling_site_elevation_in_meters"]
```

Comment:
This ended up being the only real non-string column.  Everything else made sense as a string in SQLite.

Reply:
Aren't the various `_categories` fields multi-valued? Should these be strings or a list in the resulting ssqlite?